app/src/features/apiClient/helpers/APIClientManager/index.ts

```diff
@@ -0,0 +1,46 @@
+import { makeRequest } from "../../screens/apiClient/utils";
+import { RQAPI } from "../../types";
+import { executePrerequestScript, executePostresponseScript } from "./modules/scripts/utils";
```

Comment:
This shouldn't be in utils. Its should be the entry points and should be in `index.ts`. Please note this in refractoring